### ISSUE TITLE
Clarify font license

### ISSUE THREAD

**Issue by Nateowami** [Khan/KaTeX#339]
The license clearly states that the software is under MIT license, but the font metadata says 

    Copyright (c) 2009-2010 Design Science, Inc.
    Copyright (c) 2014 Khan Academy
Since there is no mention of Design Science in the license file I'm afraid I would be incorrect to assume it's also under MIT license. Can you clarify the license and/or point me to the original license from Design Science? Issue #17 was regarding adding Khan Academy to the font license metadata, and linked to Wikipedia on [SIL Open Font License](https://en.wikipedia.org/wiki/SIL_Open_Font_License), leading me to believe they are under this license, but again, this is not clearly stated.

TL;DR: What's the license on the fonts? I'd like to use them in another project.

**Comment by xymostech** [comment]
The fonts are licensed under the SIL Open Font License, it says that in the metadata of the font but it's sorta hard to find. If you want to check using FontForge, you can look under Element -> Font Info -> TTF Names -> License, where it says:

```
Copyright (c) 2009-2010, Design Science, Inc. (<www.mathjax.org>)
Copyright (c) 2014 Khan Academy (<www.khanacademy.org>),
with Reserved Font Name KaTeX_Main.

This Font Software is licensed under the SIL Open Font License, Version 1.1.
This license available with a FAQ at:
http://scripts.sil.org/OFL
```

Hopefully that's all you need! Glad to see the fonts are being useful!

**Comment by xymostech** [comment]
(Also note that the fonts are built using a modified toolchain from the https://github.com/Khan/Mathjax-dev repo, with instructions on how to build in https://github.com/Khan/KaTeX/tree/master/dockers/MathJaxFonts)

**Comment by Nateowami** [comment]
Thank you! Sorry, I should have grabbed FontForge and looked to see if there was more metadata that `gnome-font-viewer` didn't show. For some reason I didn't think of that.

**Comment by xymostech** [comment]
No problem at all. :) We probably should make it more explicit somewhere that the fonts are under SIL...
